packages/neon/neon/test/account_test.dart

```diff
@@ -22,4 +22,42 @@ void main() {
       expect(credentials, equals(credentials));
     });
   });
+
+  group('URI', () {
+    const testURL = 'apps/test?123=456#789';
+
+    for (final (serverURL, path) in {
+      'http://localhost': '',
+      'http://localhost:443': '',
+      'http://localhost:443/nextcloud': '/nextcloud',
+    }.entries.map((final entry) => (entry.key, entry.value))) {
```

Comment:
Can you just use a list with records?

```dart
for (final (serverURL, path) in [
      ('http://localhost', ''),
      ('http://localhost:443', ''),
      ('http://localhost:443/nextcloud', '/nextcloud'),]) {
```

should also work and is way more readable

Reply:
Ah true :woman_facepalming: